src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,20 @@
-export const UserInfo = () => {};
+import './UserInfo.scss';
```

Comment:
Consider renaming the file to `UserInfoComponent.tsx` to follow naming conventions. 📝